### ISSUE TITLE
When will the next update will be available?

### ISSUE THREAD

**Issue by dfanaei** [RestyaPlatform/board#3245]
Hey Guys,
I know this is not a development issue. But I'm just wondering when the next update (Roxette) will be done?
I have been following for several weeks and have been putting this into my schedule to update the current version as it doesn't have all the features we need, but every time I come here I see the due date has been pushed back. 
Are you planning to finish it soon? Or are we expecting to wait a while?
Thanks,

**Comment by rrjanbiah** [comment]
We're aiming for perfect release and so stuck in QA cycles. Will update schedule shortly.

> it doesn't have all the features we need

What feature or ticket are you referring to?

**Comment by dfanaei** [comment]
@rrjanbiah one of the main features I am certainly looking for is the ability to put Start Date in the card and see the Date Range (Start date to Due Date) in the Calendar View. it looks like you have added this feature in v0.6.8.

**Comment by srileka** [comment]
@dfanaei Yes, we have added the date range feature in Calendar view and this feature will be available in the upcoming release v0.6.8 (Roxette)
